README.md

```diff
@@ -304,6 +304,21 @@ func main() {
 }
 ```
 
+### Cookie path
+
+By default, CSRF cookies are set on the path of the request.
+
+This can create issues, if the request is done from one path to a different path.
+
+You might want to set up a root path for all the cookies; that way, the CSRF will always work across all your paths.
+
+```
+    CSRF := csrf.Protect(
+      []byte("a-32-byte-long-key-goes-here"),
+      csrf.Path("/")
```

Comment:
It's a small nit-picky comment, but there should be a `,` at the end of this line to avoid the editor complaining

Reply:
ah :D ok. I think it's even an error